src/code/02_declarations.js

```diff
@@ -96,6 +96,9 @@
                     case '0.91.0':
                         message += 'Changelog (v0.91.0):<br/><br/> [Added]  <br/> - Feature: Thumbnails on MAL have been enlarged, with added resizing options. <br/> - Feature: "Move out of the way"-feature, which moves the video when miniMAL is opened. <br/> - Feature: "Continue watching"-links has been added to both the Overview-tab in miniMAL, and to the details-tab on MAL. <br/> - Info-bubbles has been added to the settings tab in miniMAL. <br/><br/> [Changed] <br/> - Updated the miniMAL styling. <br/><br/> [Fixed] <br/> - miniMAL-button didn\'t always appear.';
                         break;
+                    case '0.91.1':
+                        message += 'KissAnimeList (v0.91.1):<br/><br/>  [Fixed] <br/> - KAL now works with the 9anime\'s new layout';
```

Comment:
Sorry, I made a typo when I wrote it.
You should remove the word: "the".
Before: "-KAL now works with the 9anime's new layout."
After: "-KAL now works with 9anime's new layout."